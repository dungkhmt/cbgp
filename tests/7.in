30 30
30 50
17 10
22 0
5 0
25 13
8 29
16 22
19 0
10 6
15 0
20 3
27 7
11 15
10 12
1 3
16 2
22 3
4 5
19 24
17 1
26 2
17 27
23 0
19 25
22 8
29 14
19 9
21 22
16 18
28 0
12 9
5 2
19 4
13 0
13 4
22 1
23 2
24 2
19 13
19 6
23 15
14 8
22 18
26 16
21 8
19 2
28 19
12 1
16 15
12 14
22 20
1
15 15 35 5 18 5 34 6 14 7 16 7 46 0 43 2 35 2 15 1 45 1 22 7 40 1 12 9 37 2 21 6 24 3 38 8 23 2 3 0 33 7 34 3 28 2 19 6 15 6 12 8 18 4 41 2 0 0 36 3 