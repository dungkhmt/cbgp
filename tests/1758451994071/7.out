(17, 10), (22, 0), (5, 0), (25, 13), (8, 29), (16, 22), (19, 0), (10, 6), (15, 0), (20, 3), (27, 7), (11, 15), (10, 12), (1, 3), (16, 2), (22, 3), (4, 5), (19, 24), (17, 1), (26, 2), (17, 27), (23, 0), (19, 25), (22, 8), (29, 14), (19, 9), (21, 22), (16, 18), (28, 0), (12, 9), (5, 2), (19, 4), (13, 0), (13, 4), (22, 1), (23, 2), (24, 2), (19, 13), (19, 6), (23, 15), (14, 8), (22, 18), (26, 16), (21, 8), (19, 2), (28, 19), (12, 1), (16, 15), (12, 14), (22, 20), 
numberIntersection=0 
minAngle=0.13493039346604618 
sumAngle=41.5187920432039 
minEdgeLength=1.0 
sumEdgeLength=1035.6290254380976 
minDistance=0.058823529411764705 
sumDistance=42939.38303908512 

0.0000000000 0.1349303935 41.5187920432 1.0000000000 1035.6290254381 0.0588235294 42939.3830390851
0: (24, 69),
1: (69, 21),
2: (33, 20),
3: (66, 23),
4: (24, 24),
5: (26, 24),
6: (87, 0),
7: (79, 6),
8: (70, 9),
9: (29, 4),
10: (88, 0),
11: (49, 22),
12: (77, 6),
13: (21, 38),
14: (73, 8),
15: (39, 26),
16: (37, 12),
17: (74, 34),
18: (46, 9),
19: (5, 1),
20: (63, 23),
21: (63, 10),
22: (56, 9),
23: (32, 25),
24: (18, 13),
25: (21, 27),
26: (34, 16),
27: (77, 17),
28: (3, 21),
29: (72, 8),
24 69 69 21 33 20 66 23 24 24 26 24 87 0 79 6 70 9 29 4 88 0 49 22 77 6 21 38 73 8 39 26 37 12 74 34 46 9 5 1 63 23 63 10 56 9 32 25 18 13 21 27 34 16 77 17 3 21 72 8 
0: (24, 69), 1: (69, 21), 2: (33, 20), 3: (66, 23), 4: (24, 24), 5: (26, 24), 6: (87, 0), 7: (79, 6), 8: (70, 9), 9: (29, 4), 10: (88, 0), 11: (49, 22), 12: (77, 6), 13: (21, 38), 14: (73, 8), 15: (39, 26), 16: (37, 12), 17: (74, 34), 18: (46, 9), 19: (5, 1), 20: (63, 23), 21: (63, 10), 22: (56, 9), 23: (32, 25), 24: (18, 13), 25: (21, 27), 26: (34, 16), 27: (77, 17), 28: (3, 21), 29: (72, 8), 
numberIntersection=0 minAngle=0.13493039346604618 sumAngle=41.51879204315785 minEdgeLength=1.0 sumEdgeLength=1035.6290254381222 minDistance=0.058823529411764705 sumDistance=42939.38303908527 
Time taken: 18.181 s
